lib/sort-related-items.js

```diff
@@ -0,0 +1,23 @@
+const config = require('../config');
```

Comment:
I think we could do the logic of the function by using aggregations on the request.